MatrixSDK/VoIP/MXCallManager.m

```diff
@@ -958,84 +905,162 @@ - (void)transferCall:(MXCall *)callWithTransferee
-                    
-                    MXStrongifyAndReturnIfNil(self);
-                    
-                    if (room == nil)
-                    {
-                        //  could not find/create a direct room with target
-                        if (failure)
-                        {
-                            failure(nil);
-                        }
-                        return;
-                    }
+                if (consultFirst)
+                {
+                    MXWeakify(self);
                     
-                    //  place a new audio call to the target to consult the transfer
-                    [self placeCallInRoom:room.roomId withVideo:NO success:^(MXCall * _Nonnull call) {
+                    [self directCallableRoomWithUser:target.userId timeout:kMXCallDirectRoomJoinTimeout completion:^(MXRoom * _Nullable room, NSError * _Nullable error) {
                         
-                        //  mark the call with target & transferee as consulting
-                        call.callWithTransferee = callWithTransferee;
-                        call.transferee = transferee;
-                        call.transferTarget = target;
-                        call.consulting = YES;
+                        MXStrongifyAndReturnIfNil(self);
                         
-                        continueBlock(call);
-                    } failure:^(NSError * _Nullable error) {
-                        MXLogDebug(@"[MXCallManager] transferCall: couldn't call the target: %@", error);
-                        if (failure)
+                        if (room == nil)
                         {
-                            failure(error);
+                            //  could not find/create a direct room with target
+                            if (failure)
+                            {
+                                failure(nil);
+                            }
+                            return;
                         }
+                        
+                        //  place a new audio call to the target to consult the transfer
+                        [self placeCallInRoom:room.roomId withVideo:NO success:^(MXCall * _Nonnull call) {
+                            
+                            //  mark the call with target & transferee as consulting
+                            call.callWithTransferee = callWithTransferee;
+                            call.transferee = transferee;
+                            call.transferTarget = target;
+                            call.consulting = YES;
+                            
+//                            MXLogDebug(@"[MXCallManager] transferCall: call.callSignalingRoom.roomId: %@, room.roomId: %@, call.callSignalingRoom.directUserId: %@", call.callSignalingRoom.roomId, room.roomId, call.callSignalingRoom.directUserId);
```

Comment:
Do you want to keep this commented code?

Reply:
No, removed.